project/dependencies.scala

```diff
@@ -25,7 +25,10 @@ object Dependencies {
     "org.webjars" % "leaflet" % "0.7.7",
     "org.webjars" % "angular-leaflet-directive" % "0.8.2",
     "com.github.nscala-time" %% "nscala-time" % "2.10.0",
-    "org.scalactic" %% "scalactic" % "2.2.6"
+    "org.scalactic" %% "scalactic" % "2.2.6",
```

Comment:
minor comment, could you move those `webjars` next to each other that looks like a group?